lib/Twig.php

```diff
@@ -168,26 +150,20 @@ public static function maybe_convert_array( $post_id, $class ) {
 	 * Function for Term or Timber\Term() within Twig
 	 * @since 1.5.1
 	 * @author @jarednova
-	 * @param integer $tid the term ID to search for
+	 * @param integer $term_id the term ID to search for
 	 * @param string $taxonomy the taxonomy to search inside of. If sent a class name, it will use that class to support backwards compatibility
 	 * @param string $TermClass the class to use for processing the term
 	 * @return Term|array
 	 */
-	function handle_term_object( $tid, $taxonomy = '', $TermClass = 'Timber\Term' ) {
+	function handle_term_object( $term_id, $taxonomy = '', $TermClass = 'Timber\Term' ) {
 		if ( $taxonomy != $TermClass ) {
 			// user has sent any additonal parameters, process
 			$processed_args = self::process_term_args($taxonomy, $TermClass);
 			$taxonomy = $processed_args['taxonomy'];
 			$TermClass = $processed_args['TermClass'];
 		}
 
-		if ( is_array($tid) && !Helper::is_array_assoc($tid) ) {
-			foreach ( $tid as &$p ) {
-				$p = new $TermClass($p, $taxonomy);
-			}
-			return $tid;
-		}
```

Comment:
Just noticed I unintentionally removed the taxonomy arg entirely...since maybe_convert_array only currently supports one arg, should we expand it to take an array of args perhaps?